thinkhazard/templates/report.jinja2

```diff
@@ -79,7 +79,7 @@ Think Hazard - {{ division.name}}
           <h2 class="page-header detail">
             {{ current_hazard.hazardtype.title }}
             <small>
-              Hazard level: <span class="level">{{ current_hazard.categorytype.title }}</span>
+              Hazard level: <span class="level hazard-active">{{ current_hazard.categorytype.title }}</span>
```

Comment:
Is is really useful to add a new `hazard-active` class?
What about relying on the already existing level class?